models/spine/client.ts

```diff
@@ -0,0 +1,26 @@
+export interface SpineRequest {
```

Comment:
Comment for this entire file: These aren't actually modelling anything from Spine, they're the types we use internally for our Spine client. Should they even live in the models module or should they live alongside the Spine client?